package.json

```diff
@@ -104,6 +104,7 @@
     "typescript": "^4.0.3"
   },
   "dependencies": {
+    "chai-as-promised": "^7.1.1",
```

Comment:
Should this be a dependency or devDependency? See https://medium.com/@stalonadsl948/dependencies-vs-devdependencies-926e096a3dee for more info